packages/front-end/components/Metrics/MetricForm/index.tsx

```diff
@@ -311,7 +312,7 @@ const MetricForm: FC<MetricFormProps> = ({
       minPercentChange: minPercentChange / 100,
     };
 
-    if (!edit && project) {
+    if (!edit && !duplicate && project) {
```

Comment:
We also need to add `projects` to the form's defaultValues above, otherwise that field won't be copied when duplicating a metric.

```ts
projects: current.projects || []
```

Reply:
I believe this is already taken care of in the latest push, but I didn't use default form values for this (since "projects" is not actually on the form):
```
    if (duplicate) {
      sendValue["projects"] = current?.projects ?? [];
    } else if (!edit && project) {
      sendValue["projects"] = [project];
    }
```

Reply:
Now that I think about this, a project selector should probably be on that form. Will add this...